packages/forms/src/lib/wysiwyg/README.md

```diff
@@ -0,0 +1,67 @@
+# @acpaas-ui/ngx-components/forms
+
+A component to add a wysiwyg to a page or a form, build upon [ckeditor](https://ckeditor.com) and the [ng2-ckeditor](https://github.com/chymz/ng2-ckeditor) package.
+
+## Usage
+
+```typescript
+import { WysiwygModule } from '@acpaas-ui/ngx-components/wysiwyg'`;
+```
+
+## Documentation
+
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
+
+## Dependencies
+* [ckeditor](https://ckeditor.com)
+* [ng2-ckeditor](https://github.com/chymz/ng2-ckeditor)
+
+You will need to add the library to your app, preferably with a CDN.
+
+```typescript
+script: [
+        { src: 'https://cdn.ckeditor.com/4.6.2/standard/ckeditor.js' },
+        ...
+    ]
+```
+
+### API
+
+| Name        | Default value | Description |
+| ----------- | ------ | -------------------------- |
+| `@Input() additionalStyling: string;` | `-` | path to one or more css files.|
```

Comment:
`-` doesn't need backticks ;-)
Also, please write the docs in capitals, like in the other READMEs.